arch/arc/soc/quark_se_c1000_ss/Kconfig.defconfig

```diff
@@ -162,9 +162,6 @@ if I2C_QMSI
 config I2C_0
 	def_bool y
 
-config I2C_0_NAME
-	default "I2C_2"
-
 config I2C_0_IRQ_PRI
```

Comment:
I am sure that I am missing something here. There is no interrupt property in intel,qmsi-ss-i2c but there is IRQ_PRI here.
Same for I2C_1.